src/webui/www/private/scripts/dynamicTable.js

```diff
@@ -2177,13 +2177,13 @@ window.qBittorrent.DynamicTable = (function() {
                 let img_path;
                 switch (row.full_data.status) {
                     case 'default':
-                        img_path = 'icons/application-rss+xml.svg';
+                        img_path = 'icons/application-rss.svg';
                         break;
                     case 'hasError':
                         img_path = 'icons/task-reject.svg';
                         break;
                     case 'isLoading':
-                        img_path = 'images/spinner.gif';
+                        img_path = 'icons/loading.svg';
```

Comment:
What's the reason for this change?

Reply:
We don't even have `spinner.gif` file anymore. Does it make sense to keep this?

Reply:
> We don't even have `spinner.gif` file anymore. Does it make sense to keep this?

Yes, we do....

https://github.com/qbittorrent/qBittorrent/blob/master/src/webui/www/private/images/spinner.gif

Reply:
My mistake. Reverting the changes. 


> What do others think about the `WebUI images` folder??

I think there should be one unified folder for all image, and it should have SVG only, if possible.

Reply:
>What do others think about the WebUI images folder??

There was a consensus that webui should be self-contained. That is, clear separation of GUI and webui icons, even if some icons would need to be duplicated. This would simplify the webui server code and allows us to replace webui without worrying too much about its assets.